es/GOVERNANCE.md

```diff
@@ -1,6 +1,6 @@
 # Gobernanza
 
-Este estándar es el núcleo de la tutela o stewardship de codebases que ofrece la [Foundation for Public Code](https://publiccode.net/). Decidimos si una codebase está lista para el codesarrollo de la comunidad basándonos en este documento.
+Este estándar es el núcleo de la tutela o dirección de codebases que ofrece la [Foundation for Public Code](https://publiccode.net/). Decidimos si una codebase está lista para el codesarrollo de la comunidad basándonos en este documento.
```

Comment:
This is an interesting one. It's very difficult to find an exact translation for "stewardship", but we do not understand it as "management" or "direction". That's why we'd rather to stick to the original version.